PythonVisualizations/SimpleQueue.py

```diff
@@ -236,6 +270,15 @@ def updateNItems(self, newNItems):
         self.canvas.itemconfigure(
             self.nItemsDisplay, text='nItems: {}'.format(self.nItems))
 
+    # calculate the coordinates of the output box
+    def outputBoxCoords(self):
+        size = 20
+        padding = 30
+        return (padding,
+                padding,
+                padding+size*2,
+                padding+size*2)
+
```

Comment:
The output box should be large enough to hold the longest possible key that can be stored in the queue.   That's controlled by the `maxArgWidth` attribute of class.  There's a fairly new addition in VisualizationApp that provides the width in pixels of a string written in a particular font, so I think result of `outputBoxCoords()` should be:

```
widest = self.textWidth(self.valueFont, 'W' * self.maxArgWidth)
return (padding, padding, padding * 2 + wideest, padding + 2 * abs(self.valueFont[1]))
```
This finds out how wide the string 'WWWWWW' is (with the default 6 characters for width) and adds padding around it.  The `abs(self.valueFont[1])` gets the height in pixels of the font, so twice the height provides some padding.